drivers/hmis/app/graphql/mutations/create_form_rule.rb

```diff
@@ -0,0 +1,32 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class CreateFormRule < BaseMutation
+    argument :definition_id, ID, required: true
+    argument :input, Types::Admin::FormRuleInput, required: true
+
+    field :form_rule, Types::Admin::FormRule, null: false
+
+    def resolve(definition_id:, input:)
+      raise 'not allowed' unless current_user.can_configure_data_collection?
+
+      definition = Hmis::Form::Definition.find(definition_id)
+
+      instance = Hmis::Form::Instance.new(definition_identifier: definition.identifier)
+      instance.assign_attributes(input.to_attributes)
+
+      if instance.valid?
+        instance.save!
+        { form_rule: instance }
```

Comment:
Micro-optimization here, rails save() calls validate() and returns true/false if validation fails. So you can say:
```ruby
      if instance.save
          { form_rule: instance }
```